source/ada/lsp-ada_documents.adb

```diff
@@ -2053,7 +2055,51 @@ package body LSP.Ada_Documents is
       use type LSP.Messages.Search_Kind;
 
       procedure Refresh_Symbol_Cache;
-      procedure Insert (Item : Name_Information);
+
+      procedure Insert
+        (Item : Name_Information;
+         Name : Libadalang.Analysis.Defining_Name);
```

Comment:
since this is not an "obvious" insert (it has some intelligence), it'd be worthwhile adding a bit of doc